spec/src/main/asciidoc/core/beanmanager_lite.asciidoc

```diff
@@ -0,0 +1,268 @@
+[[programmatic_access]]
+
+== Programmatic access to container
+
+CDI 3.0 used to provide a single `BeanManager` object, which allows access to many useful operations.
+In CDI 4.0, this is split into `BeanManagerLite` and `BeanManager`.
+
+`BeanManagerLite` provides access to a subset of `BeanManager` features which can be implemented in more restricted environments;
+It is available in {cdi_lite} environments.
+
+`BeanManager` extends `BeanManagerLite` and provides the remaining features.
+It is available in {cdi_full} environments.
+
+In {cdi_lite} environment, attempting to obtain a `BeanManager` or invoking methods on it results in non-portable behavior.
+
+[[beanmanager]]
+
+=== The `BeanManagerLite` object
+
+The interface `jakarta.enterprise.inject.spi.BeanManagerLite` provides operations for obtaining contextual references for beans, along with many other operations of use to applications.
```

Comment:
Can BeanManagerLite be looked up via jndi lookup at runtime?

Reply:
I believe we should avoid references to JNDI, as well as other Jakarta EE concepts, in Lite.

Reply:
(In fact, I briefly thought we should move that JNDI thing to the EE section. I don't know why it's present in "Core CDI", it's clearly EE only and for example can't be used in SE.)

Reply:
I think I raised this question at some earlier comment on the BM Lite PR as well. Technically speaking you can get the BML via JNDI simply because you can get ordinary BM (which extends BML).

That being said, IMO we don't want any specific notion of Lite supporting JNDI in any way.